DESIGN.md

```diff
@@ -158,9 +158,9 @@ GET /login?connection=$connectionToken&localDevice=$localDeviceUUID HTTP/1.0
    json
 ~~~ json
 {
-  "localPlayerToken" : UUID,
```

Comment:
It's confusing, because the markdown indicates it's a valid json block; probably the right fix is remove the json indicator, and have a convention for annotating types, as opposed to what appears to be a literal. That comment would apply to all the json markdown I've had a go at in this PR.

Reply:
OK, so perhaps xxxxxxxx-xxxx-Mxxx-Nxxx-xxxxxxxxxxxx for UUID values